cvat-ui/src/components/cvat-app.tsx

```diff
@@ -10,6 +10,7 @@ import notification from 'antd/lib/notification';
 import Spin from 'antd/lib/spin';
 import Text from 'antd/lib/typography/Text';
 import GlobalErrorBoundary from 'components/global-error-boundary/global-error-boundary';
+import LoginWithTokenComponent from './login-with-token/login-with-token';
```

Comment:
Please, use absolute import: ``components/login-with-token/login-with-token``, or put the line after all absolute (linter says that relative import should occur after absolute)